cpp/test/Ice/optional/Test.ice

```diff
@@ -294,21 +291,11 @@ interface Initial
     ["marshaled-result"] optional(1) StringIntDict opMDict2(optional(2) StringIntDict p1,
                                                             out optional(3) StringIntDict p2);
 
-    ["marshaled-result"] optional(1) G opMG1();
-    ["marshaled-result"] optional(1) G opMG2(optional(2) G p1, out optional(3) G p2);
-
     bool supportsRequiredParams();
 
     bool supportsJavaSerializable();
 
     bool supportsCsharpSerializable();
-
```

Comment:
Should also remove supportsCsharpSerialize above. We dropped support for C# serialization a while ago.

Reply:
and supportsRequiredParams. I can't find any call to this operation.

Reply:
Yes, this is _only_ of the Java mapping.
Java is the only language which returns `true` for this, and it's also the only language that calls it.
At the top of `Ice/Optional/AllTests.java`

That's part of the Java cleanup I mentioned in the top description.